<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SBMECoordinateRegistrationDlg</class>
 <widget class="QDialog" name="SBMECoordinateRegistrationDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Rx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>311</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rx:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ry">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>311</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ry:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Rz">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>311</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Tx">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>354</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tx:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Ty">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>354</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ty:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Tz">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>354</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetNDIPoint">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get NDI Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetModelPoint">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Model Point</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_StartRegister">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Rx">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Ry">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Rz">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Tx">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Ty">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_TZ">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Info">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NDIpointX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NDIpointY">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NDIpointZ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ModelPointX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ModelPointY">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ModelPointZ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ModelPointX">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>151</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ModelPointX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ModelPointY">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>151</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ModelPointY:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ModelPointZ">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>151</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ModelPointZ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NDIPointX">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>51</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NDIPointX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NDIPointY">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>51</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NDIPointY:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NDIPointZ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>51</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NDIPointZ:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
